Step-by-step-Tutorials/Unit-Testing-Grains.md

```diff
@@ -69,7 +69,7 @@ namespace Tests
 
 ```
 
-As you can see having `ClassCleanup()` is optional but it's good to have it around. 
+Make sure to add `async` to the generated test method for the test to be recognized by Visual Studio. As you can see having `ClassCleanup()` is optional but it's good to have it around. 
```

Comment:
not sure I follow the reasoning to why adding "async" makes it recognizable by VS. Or did you mean to say something like:
Since this test method is asynchronous and leverages the `await` keyword, be sure the method is defined as `async` and returns a `Task`.